examples/federated-learning/func_lib.py

```diff
@@ -35,7 +35,7 @@ def reptile_model_fn(data_owner, iterations=3,
         zip(grads_k, data_owner.model.trainable_variables),
     )
 
-  return model.trainable_variables
+  return [var.read_value() for var in data_owner.model.trainable_variables]
```

Comment:
why is the `read_value` needed? I vaguely remember that they changed the semantics in TF2 to not require this?

Reply:
Its just because this function is wrapped in a local computation and is expecting a tensor to be returned. Without `read_value` a ResourceVariable is returned. We could fix it in the local computation implementation but this works to.